johnsonandjohnson/Bodiless-JS
# fix(component PluginButton): Additional ... in RTE throws type is invalid error and serves WSOD (#536)

## Changes
Fix the WSOD when additional styles are are added to the Rich Text Editor.

## Test Instructions
Steps to reproduce

1. Configure the RTE to serve additional styles so you get the ... (Additional Styles button)
2. Edit https://github.com/johnsonandjohnson/Bodiless-JS/blob/master/examples/test-site/src/components/Editors/index.tsx#L81
3. add sample styles:
   CenterHeader: flow(asHeader1, asAlignCenter, asItalic),
   UnderlineRightHeader: flow(asHeader1, asAlignRight, asUnderline),
4. Bring up Site, go to RTE field that uses full editor, be in edit mode.
5. Click the ... in Rich Text Editor Toolbar
Expected result
The additional styles are shown without error on Console
## Related Issues
Fixes #536
